_maps/shuttles/shiptest/pill_super.dmm

```diff
@@ -0,0 +1,290 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/obj/machinery/atmospherics/pipe/heat_exchanging/manifold4w/layer2,
+/obj/machinery/atmospherics/pipe/simple/general/visible{
+	dir = 5
+	},
+/obj/machinery/atmospherics/pipe/simple/general/visible/layer4{
+	dir = 5
+	},
+/obj/machinery/portable_atmospherics/canister/toxins,
+/obj/machinery/atmospherics/components/unary/portables_connector/visible/layer1{
+	dir = 4
+	},
+/obj/structure/catwalk,
+/turf/open/floor/plating/rust,
+/area/ship/storage)
+"b" = (
+/obj/machinery/atmospherics/pipe/heat_exchanging/junction/layer4{
+	dir = 4
+	},
+/obj/machinery/atmospherics/pipe/heat_exchanging/manifold4w/layer2,
+/obj/machinery/atmospherics/pipe/simple/general/visible{
+	dir = 4
+	},
+/obj/machinery/atmospherics/components/binary/pump/on/layer1{
+	dir = 4
+	},
+/obj/structure/catwalk,
+/turf/open/floor/plating/rust,
+/area/ship/storage)
+"c" = (
+/obj/machinery/atmospherics/pipe/heat_exchanging/manifold4w/layer4,
+/obj/machinery/atmospherics/pipe/heat_exchanging/manifold4w/layer2,
+/obj/machinery/atmospherics/pipe/heat_exchanging/junction{
+	layer = 3;
+	dir = 4
+	},
+/obj/machinery/atmospherics/pipe/simple/general/visible/layer1{
+	dir = 9
+	},
+/obj/docking_port,
+/obj/structure/catwalk,
+/turf/open/floor/plating/rust,
+/area/ship/storage)
+"f" = (
+/obj/machinery/power/emitter/welded{
+	dir = 4
+	},
+/obj/structure/cable{
+	icon_state = "0-2"
+	},
+/obj/machinery/atmospherics/pipe/heat_exchanging/junction/layer2{
+	dir = 4;
+	layer = 3
+	},
+/obj/machinery/atmospherics/pipe/simple/general/visible/layer1{
+	dir = 10
+	},
+/obj/structure/catwalk,
+/turf/open/floor/plating/rust,
+/area/ship/storage)
+"i" = (
+/obj/machinery/power/supermatter_crystal{
+	layer = 3.4
```

Comment:
could you make this a code change 🥺

Reply:
You just want me to make another pr that puts all sms on the 3.4 layer by default?

Reply:
Replaced the crystal with a shard, still gotta make the higher layer for the SM it's own PR